pym/bob/invoker.py

```diff
@@ -509,14 +510,26 @@ async def runCommand(self, args, cwd=None, **kwargs):
         ret = await self.__runCommand(args, cwd, **kwargs)
         return ret
 
-    async def callCommand(self, args, cwd=None, **kwargs):
+    async def callCommand(self, args, cwd=None, retries=0, success=0, **kwargs):
         cwd = os.path.join(self.__cwd, cwd) if cwd else self.__cwd
-        ret = await self.__runCommand(args, cwd, **kwargs)
+        while True:
+            ret = await self.__runCommand(args, cwd, **kwargs)
+            if (ret == success) or (retries == 0):
+                break
+            time.sleep(3)
```

Comment:
This is an async function. You need to use 
```suggestion
            await asyncio.sleep(3)
```